src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -34,18 +34,29 @@ public class DeleteCommand extends Command {
             + "Parameters: n/Name or id/Nric (must be valid)\n"
             + "Example: " + COMMAND_WORD + " n/John Doe or " + COMMAND_WORD + " id/S1234567A";
 
-    public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Deleted Person: %1$s";
+    public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Deleted Patient: %1$s";
 
-    public static final String MESSAGE_DELETE_PERSON_FIELD_SUCCESS = "Deleted Person's field: %1$s";
+    public static final String MESSAGE_DELETE_PERSON_FIELD_SUCCESS = "Deleted Patient's field: %1$s";
 
     public static final String MESSAGE_PERSON_NOT_FOUND =
-            "The given combination of Name and NRIC does not match any person in the patients list.";
+            "The given combination of Name and NRIC does not match any patient in the patients list.";
 
-    private final DeletePersonDescriptor deletePersonDescriptor;
+    public static final String MESSAGE_UNDO_DELETE_ENTIRE_PERSON_SUCCESS = "Undoing the deletion of Patient:  %1$s";
```

Comment:
I think it will be better to name the variable `MESSAGE_UNDO_DELETE_PERSON_SUCCESS` so as to correspond with the variable name when a patient is successfully deleted.